phys/module_radiation_driver.F

```diff
@@ -166,6 +167,7 @@ SUBROUTINE radiation_driver (                                          &
               ,aod5502d, angexp2d, aerssa2d, aerasy2d                     &
               ,aod5503d                                                   &
               ,cw_rad, shcu_physics                                       &
+              ,obscur,mask,elat_track,elon_track			                 & ! amontornes-bcodina 2015/09 solar eclipses
```

Comment:
@irowebbn These four variables, along with namelist sw_eclipse from above should be passed into radiation driver from solver.F, but these variables are missing from the call to radiation_driver in dyn_em/module_first_rk_step_part1.F.

Reply:
I found these variables as well as namelist sw_eclipse in the file module_first_rk_step_part1.F, but why aren't they show up in the modified files?
Also right now, elat_track and elon_track are just single values dumped in wrfout file. Is that ok?